BlueDBOnDisk/src/main/java/org/bluedb/disk/Blutils.java

```diff
@@ -165,4 +165,20 @@ public static Runnable surroundTaskWithTryCatch(Runnable r) {
 			}
 		};
 	}
+	
+	public static void log(String message) {
+		System.out.println(String.format("%1$TF %1$TT.%1$TL %2$s", System.currentTimeMillis(), message));
+	}
+	
+	public static String getStackTraceAsString() {
+		StringBuilder sb = new StringBuilder();
+		
+		StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
+	    for(int i = 1; i < stackTrace.length; i++) {

```

Comment:
I am actually starting at index 1 instead of 0 so that I can skip the first element. It is always Thread.getStackTrace() which I don't really care to see every time.